src/main/java/org/broadinstitute/hellbender/utils/read/ReadConstants.java

```diff
@@ -0,0 +1,17 @@
+package org.broadinstitute.hellbender.utils.read;
+
+/**
+ * Constants for use with the GATKRead interface
+ */
+public final class ReadConstants {
```

Comment:
Putting constants in an interface seems to be frowned upon by the Java community